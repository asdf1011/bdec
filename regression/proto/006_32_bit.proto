message fixed32types {
    required fixed32 a = 1;
    required sfixed32 b = 2;
    required float c = 3;
}
