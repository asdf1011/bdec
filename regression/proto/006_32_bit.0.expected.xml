<fixed32types>
    <a>0</a>
    <b>-2147483648</b>
    <c>3.14159274101</c>
</fixed32types>
